### PR TITLE
Code improvement and add new mcdev command logic

### PR DESCRIPTION
**Pull request by anasilva105**
# PR details

## What changes did you make? (Give an overview)

Base code restructured and improved and new mcdev command logic implemented. Also jsdoc were added to the code.

### CODE REVIEW

File: .eslintrc
Comment (JoernBerkefeld) [inline .eslintrc:1]: Have you seen that they moved away from this json format and towards a js format recently?

Mcdev already uses that

File: package.json [hunk 1]
```diff
@@ -5,9 +5,6 @@
    "version": "2.1.1",
    "license": "MIT",
    "publisher": "Accenture-oss",
-   "peerDependencies": {
-      "mcdev": ">=5.2.0"
```
Comment (JoernBerkefeld) [inline package.json:9]: Why are removing that? Im sure you told me but I can't remember

File: src/devtools/main.ts
Comment (JoernBerkefeld) [inline src/devtools/main.ts:1]: Empty file?

File: src/messages/devtools.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
+const noMcdevInstalled =
+	"It appears that DevTools package 'mcdev' is not installed on your system. Would you like to install it?";
```
Comment (JoernBerkefeld) [inline src/messages/devtools.ts:2]: ❤️

File: package.json (hunk 1 above)
Comment (anasilva105) [reply package.json:9]: I added it again after noticing that for some reason it was removed. I guess this branch didn't have it. You can see it now on line 16-18

File: src/devtools/main.ts
Comment (anasilva105) [reply src/devtools/main.ts:1]: This one and some other files were deleted.

**Review by JoernBerkefeld — APPROVED** [review]

**Pull request merged**